lang/en/validation.php

```diff
@@ -408,6 +408,7 @@
     'study_rate_required' => 'Study rate is required.',
     'study_rate_numeric' => 'Study rate must be a number.',
     'graduation_date_required' => 'Graduation date is required.',
+    "access-denied"=>"access denied",
```

Comment:
The string `access-denied` should follow the same naming convention as other keys in this file (snake_case). Suggested change:
```suggestion
    'access_denied' => 'access denied',
```